unit_test/test_spdm_responder/encap_challenge.c

```diff
@@ -100,6 +100,7 @@ void libspdm_test_responder_encap_challenge_case2(void **state)
     size_t response_size;
     void *data;
     size_t data_size;
+    bool need_continue;
```

Comment:
why we need change test case?

need_continue should not be assigned if the status is ERROR